src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts

```diff
@@ -270,6 +276,25 @@ export default class AdyenV2PaymentStrategy implements PaymentStrategy {
         });
     }
 
+    private _handleFromAction(action: AdyenV2Action, paymentMethodId: string): Promise<any> {
+        return new Promise(() => {
+            if (!this._adyenCheckout) {
+                throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
+            }
+
+            if (paymentMethodId === AdyenV2PaymentMethodType.IDEAL) {
+                const { onLoad } = this._getAdyenV2PaymentInitializeOptions().threeDS2Options;
+                const adyenComponent = this._adyenCheckout
+                    .createFromAction(action);
+                const container = this._getAdyenV2PaymentInitializeOptions().threeDS2ContainerId;
```

Comment:
Are you trying to support a 3DS flow? It doesn't seem to be - you're trying to support online bank transfer instead. But here, you're using the container ID and options for 3DS flow? I don't think that's appropriate, you either use the default options and `onload` callback or define something new for this flow.

Reply:
Both 3DS1 or online back transfers are full redirects asking for additional info and a 3DS2 challenge shopper. As you can see, all have in common one thing, they all are waiting for additional info to be captured.

My suggestion is to simply rename the container to something more generic like `needs-action-container-id` be able to mount every component where additional info is required.

Reply:
i.e. `scheme-adyen-component-field-additional-action`, or `ideal-adyen-component-field-additional-action`

Reply:
In fact, I'll rather suggest to switch `payment method`and `payment provider` order to have something like `adyen-scheme-additional-action-component-field`